src/constraints/calculate_constraint_output.hpp

```diff
@@ -0,0 +1,92 @@
+#pragma once
+
+#include <Kokkos_Core.hpp>
+
+#include "constraint_type.hpp"
+
+#include "src/math/quaternion_operations.hpp"
+#include "src/math/vector_operations.hpp"
+#include "src/types.hpp"
+
+namespace openturbine {
+
+struct CalculateConstraintOutput {
```

Comment:
Is there a better name here than "output"?  Perhaps `CalculateConstraintAngularMotion` or something along those lines?

Reply:
Possibly, but I could see a possibility of having output for other constraint types, so I am leaving it general until I rule that out.

Reply:
Sounds good.